common/dbus/platform/asio_libdbus/dbus.cpp

```diff
@@ -578,6 +578,178 @@ optional::optional<DBusMethodHandler<ExpectedStringPair>> DBusObject::GetMethodH
 	}
 }
 
+error::Error DBusServer::InitializeConnection() {
+	auto err = DBusPeer::InitializeConnection();
+	if (err != error::NoError) {
+		return err;
+	}
+
+	DBusError dbus_error;
+	dbus_error_init(&dbus_error);
```

Comment:
For future safeness, I would wrap this in a smart pointer

Reply:
I know you would. :wink: